File: php/telegram_channel_bot.php
<?php
require_once('telegram_config.php');
function send_post($txt){
	$url = "https://api.telegram.org/bot".TelegramBotConfig::$bot_token."/sendMessage";
	$url .= "?chat_id=".TelegramBotConfig::$channel_id."&parse_mode=".TelegramBotConfig::$parse_mode."&disable_web_page_preview=true&text=$txt";
	$res = file_get_contents($url);
	if($res){
		if(TelegramBotConfig::$show_output){
			print_r(json_decode($res, TRUE));
		}	
	}
	else{
		echo "Error sending the post";
	}
}

File: python/telegram_channel_bot.py
#!/your_path_to_python
# -*- coding: utf8 -*-

import requests
import telegram_config
import json

def send_post(txt):
    url = "https://api.telegram.org/bot{}/sendMessage"
    u = url.format(telegram_config.bot_token)
    p = {'chat_id':telegram_config.channel_id,'parse_mode':'html','disable_web_page_preview':True, 'text':txt}
    r = requests.post(u,data=p)
    if r.status_code == 200:
        if telegram_config.show_output:
            print(json.dumps(r.json(), sort_keys=True, indent=4, separators=(',', ': ')))
    else:
        print("Error sending post: {}\n{}".format(r.status_code, r.text))

